tests/test-extensions/metadata-extension/src/main/java/org/mule/test/metadata/extension/MetadataOperations.java

```diff
@@ -276,6 +278,15 @@ public String returnQuery(@MetadataKeyId String query) {
     return query;
   }
 
+  @Query(translator = MetadataExtensionQueryTranslator.class,
+      entityResolver = MetadataExtensionEntityResolver.class,
+      nativeOutputResolver = NativeQueryOutputResolver.class)
+  @MediaType(TEXT_PLAIN)
+  public String queryWithOtherMetadata(@MetadataKeyId(QueryExtraResolver.class) String query,
```

Comment:
I think that if you use DSQL, you cannot have input resolvers nor metadata key resolvers. You are adding the to the TypeResolversModelProperty, but they are not in the ResolversFactory.